extension/data/guiobject/guiobject.html

```diff
@@ -2,7 +2,8 @@
 <html>
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-  <link rel="stylesheet" href="widget.css" type="text/css" />
+  <link rel="stylesheet" href="guiobject.css" type="text/css" />
+  <script type="application/javascript;version=1.8" src="guiobject.js"></script>
```

Comment:
I chose that name because it's used by `Widget` and `Frame` modules.  It's already reflects where it is used. 
There is no toolbar here. The `Frame` is inlined then into `Toolbar`.

Also if Issue  #176 got fixed, it would be nice to reuse this file set.

Sure this is not the best name, but better than `toolbar.*` ....
What about: `reportingPanel.*`, `reportingFrame.*`, `dataPanel.*`?